packages/tools/grub/package.mk

```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="grub"
-PKG_VERSION="6a34fdb76a07305b95e31659bc27b1d190101cbf"
-PKG_SHA256="825328841e2e7e8654518808965afe19a421f6e272d946a5fe11bd90b8582f76"
+PKG_VERSION="2df291226638261d50fadcab1f5edb6c12ab6cfd"
```

Comment:
it looks to me like the latest version is `2.29.2` I'm not sure why we are using git snapshots anyways.

Reply:
latest grub is 2.04
the current git version `6a34fdb76a07305b95e31659bc27b1d190101cbf` (2.04+) is used due gcc10 compatibility, so updating sounds reasonable (no 2.05 yet)

Reply:
correct, I'm not sure where I got `2.29.2` from....